From a751e554a906a8d21eee4be0d1ffe98f2a7f0247 Mon Sep 17 00:00:00 2001
From: Mikolaj Izdebski <mizdebsk@redhat.com>
Date: Mon, 20 Oct 2014 15:08:43 +0200
Subject: [PATCH 02/14] Use Maven Wagon 2.8

---
 gradle/dependencies.gradle | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/gradle/dependencies.gradle b/gradle/dependencies.gradle
index 19d3ad0..14a360f 100755
--- a/gradle/dependencies.gradle
+++ b/gradle/dependencies.gradle
@@ -125,9 +125,9 @@ libraries.maven3 = dependencies.module("org.apache.maven:maven-core:3.0.4") {
     dependency "org.apache.maven:maven-plugin-api:3.0.4@jar"
     dependency "org.apache.maven:maven-aether-provider:3.0.4@jar"
 
-    dependency 'org.apache.maven.wagon:wagon-http:2.4@jar'
-    dependency 'org.apache.maven.wagon:wagon-provider-api:2.4@jar'
-    dependency 'org.apache.maven.wagon:wagon-http-shared4:2.4@jar'
+    dependency 'org.apache.maven.wagon:wagon-http:2.8@jar'
+    dependency 'org.apache.maven.wagon:wagon-provider-api:2.8@jar'
+    dependency 'org.apache.maven.wagon:wagon-http-shared:2.8@jar'
 
     //eather:
     dependency "org.sonatype.aether:aether-api:1.13.1@jar"
-- 
2.1.0

