From 291e2fb748f6927eae4ffead739bf223dd871063 Mon Sep 17 00:00:00 2001
From: Mikolaj Izdebski <mizdebsk@redhat.com>
Date: Fri, 7 Nov 2014 15:02:11 +0100
Subject: [PATCH 05/14] Use jansi 1.11

---
 subprojects/native/native.gradle | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/subprojects/native/native.gradle b/subprojects/native/native.gradle
index 1639a64..96e46cc 100755
--- a/subprojects/native/native.gradle
+++ b/subprojects/native/native.gradle
@@ -7,7 +7,7 @@ dependencies {
     compile libraries.slf4j_api
     compile libraries.jna
     compile libraries.nativePlatform
-    compile module('org.fusesource.jansi:jansi:1.2.1') {
+    compile module('org.fusesource.jansi:jansi:1.11') {
         dependency libraries.jna
     }
     compile libraries.guava
-- 
2.1.0

